ADAL/src/cache/ADTokenCacheItem+Internal.m

```diff
@@ -40,24 +40,24 @@ @implementation ADTokenCacheItem (Internal)
 - (void)checkCorrelationId:(NSDictionary*)response
       requestCorrelationId:(NSUUID*)requestCorrelationId
 {
-    AD_LOG_VERBOSE(@"Token extraction", requestCorrelationId, @"Attempt to extract the data from the server response.");
+    AD_LOG_VERBOSE(requestCorrelationId, nil, NO, @"Token extraction. Attempt to extract the data from the server response.");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 126 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=objc%3ALineLength)